driver/modern_bpf/programs/tail_called/events/syscall_dispatched_events/access.bpf.c

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (C) 2022 The Falco Authors.
+ *
+ * This file is dual licensed under either the MIT or GPL 2. See MIT.txt
+ * or GPL2.txt for full copies of the license.
+ */
+
+#include <helpers/interfaces/fixed_size_event.h>
+#include <helpers/interfaces/variable_size_event.h>
+
+/*=============================== ENTER EVENT ===========================*/
+
+SEC("tp_btf/sys_enter")
+int BPF_PROG(access_e,
+	     struct pt_regs *regs,
+	     long id)
+{
+	struct auxiliary_map *auxmap = auxmap__get();
```

Comment:
Also wonder whether to use `ringbuf__reserve_space` here instead of the auxmap as fixed-sized event data are pushed. @Andreagit97 is there some doc availablen when to to use/prefer the one or other?